.gitignore

```diff
@@ -29,4 +31,4 @@ __MACOSX
 
 brainiak/fcma/cython_blas.c
 brainiak/eventseg/_utils.c
-brainiak/examples/fcma/face_scene/
+examples/fcma/face_scene/
```

Comment:
I can see a lot of lines like this showing up over time. Please remove it and keep your datasets outside of BrainIAK if you prefer them not to show up in `git status`.